staging/src/k8s.io/apiserver/pkg/authentication/request/websocket/protocol.go

```diff
@@ -24,8 +24,8 @@ import (
 	"strings"
 	"unicode/utf8"
 
+	"k8s.io/apimachinery/pkg/util/httpstream/wsstream"
```

Comment:
This auth fine change is trivial, ping me whenever the overall PR is ready for approval.

Reply:
The PR is ready for approval. Thanks Mo.